### PR TITLE
Beginning of banner and added pagination

### PR DESCRIPTION
**Pull request by isabelsang**
Want to center text and make the photo be in the background. Also ideally want to have the width of the banner spread to the edges.
image

Picture got cut off at bottom but the pagination doesn't actually get cut off
image

### CODE REVIEW

File: frontend/src/components/dashboard/Banner.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  bannerImg: {
+    background: 'linear-gradient((rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5))',
+  },
+}));
+
+export default function Banner({title, description, imgURL}) {
+  const classes = useStyles();
+
+  return (
+     //<div className={classes.banner} style={{ backgroundImage: 'linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url(${imgURL})'}}>
```
Comment (katch00) [inline frontend/src/components/dashboard/Banner.js:30]: delete commented code in file

File: frontend/src/components/dashboard/Dashboard.js [hunk 2]
```diff
@@ -157,12 +159,11 @@ export default function Dashboard() {
...
-                <Typography variant="h3" gutterBottom>
-                  Museum Gallery
-                </Typography>
-                <Typography variant="body1" gutterBottom>
-                  Explore the Art Institute of Chicago!
-                </Typography>
+                <Banner
+                    title="Museum Gallery"
+                    description="Explore the Art Institute of Chicago!"
+                    imgURL={AICimg}
```
Comment (katch00) [inline frontend/src/components/dashboard/Dashboard.js:165]: style: indentation

File: frontend/package.json [hunk 3]
```diff
@@ -44,10 +45,5 @@
   "engines": {
     "node": "10.x.x"
   },
-  "devDependencies": {
```
Comment (indhu-gunda) [inline frontend/package.json:47]: I only needed to take out eslint to fix the problem. I think it's still a good idea to have to have the others.

File: frontend/src/components/dashboard/Gallery.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { fade, makeStyles } from '@material-ui/core/styles';
+import { makeStyles } from '@material-ui/core/styles';
```
Comment (indhu-gunda) [inline frontend/src/components/dashboard/Gallery.js:2]: style nit: there shouldn't be space between the brackets and makeStyles

File: frontend/src/components/dashboard/Banner.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+const useStyles = makeStyles((theme) => ({
+  banner: {
+    backgroundColor: 'rgb(236,227,218)',
+    backgroundPosition: 'center',
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    // height: '300px',
```
Comment (BearBurg) [inline frontend/src/components/dashboard/Banner.js:14]: delete this line?

Comment (isabelsang) [reply frontend/src/components/dashboard/Banner.js:14]: Resolved in next PR

**Pull request merged**